packages/client/components/DashNavList/DashNavList.tsx

```diff
@@ -43,11 +44,11 @@ const DashNavList = (props: Props) => {
     organizationsRef
   )
 
-  const tiers = ['enterprise', 'team', 'starter']
+  const TierEnumValues: TierEnum[] = ['enterprise', 'team', 'starter']
```

Comment:
@nickoferrall does this work to check the tiers?

Reply:
See [commit](https://github.com/ParabolInc/parabol/pull/9973/commits/ff294be2931c17ecb3832b55e1762113ebe88d14)

Reply:
That works!